app/views/membership.html

```diff
@@ -125,7 +125,7 @@ <h3>
                       ng-model="subject.newRole"
                       theme="bootstrap"
                       search-enabled="true"
-                      title="Select a new role for {{subjectKind.name}}"
+                      title="Select a new role for {{subject.name}}"
```

Comment:
Also, this change is clearer:

![screen shot 2018-01-03 at 11 21 51 am](https://user-images.githubusercontent.com/280512/34528896-546aa53a-f078-11e7-8673-12f801917680.png)